.github/workflows/ci.yaml

```diff
@@ -12,13 +12,12 @@ jobs:
     strategy:
       matrix:
         ruby-version:
-        - '2.5'
-        - '2.6'
         - '2.7'
         - '3.0'
```

Comment:
Should this also add ruby 3.1?

Reply:
yeah, I was debating it in all my prs and didn't in some.  I can add it.  I should probably circle around and do it for all of them that are standalone.

Reply:
good call, tests are broken on ruby 3.1

Reply:
fixed